### DISCUSSION TITLE
View EventData in the Message Column

### DISCUSSION THREAD

**Discussion by JodyBevan** [kmaki565/EventLook#45] [Q&A]
The application I used to develop with has an interface to send to the macOS Console, and the Windows Event Log. We can send our custom message. This shows up in the XML in the EventData.

This is an example of the EventData
<?xml version="1.0" encoding="utf-16"?>
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="4D Application" />
    <EventID Qualifiers="49152">7</EventID>
    <Version>0</Version>
    <Level>2</Level>
    <Task>0</Task>
    <Opcode>0</Opcode>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2023-06-06T07:30:19.6857157Z" />
    <EventRecordID>35166</EventRecordID>
    <Correlation />
    <Execution ProcessID="34820" ThreadID="0" />
    <Channel>Application</Channel>
    <Computer>Manuel_Privado</Computer>
    <Security />
  </System>
  <EventData>
    <Data>                                            Deleted Files Folder does not exist at: C:\Users\villa\Desktop\IP test for Jody with logs\Infopool\DeletedFiles\</Data>
  </EventData>
</Event>

The Message column shows for all of these:
(EventLook) The description for the event cannot be found.

How do I get the Message column to show the text message in EventData?

I am currently using EventLook 1.1.1.0 (5/22/2023) that I downloaded from the Microsoft Store. I am using Windows 10.

**Comment by JodyBevan** [comment]
Further to my question. The file I am looking at me was sent by our support department for me to examine. I add this just in case it matters. 

The file is an .evtx file (by the extension)

**Comment by kmaki565** [comment]
Resolved by #49
